charts/devnet/templates/chains/cosmos/genesis.yaml

```diff
@@ -51,6 +51,14 @@ spec:
     spec:
       {{- include "imagePullSecrets" $chain | indent 6 }}
       initContainers:
+        {{- if hasKey $chain "storage" }}
```

Comment:
FYI this would have to be added to `validator.yaml` file as well and not just genesis.